client/src/components/AssetTable/AssetRow/AssetRow.js

```diff
@@ -76,6 +76,8 @@ function AssetRow(props) {
   async function handleSelect() {
     //on click, table rows should highlight and add themselves to the selected asset list
     if (asset.checked_out) {
+      setAlertMessage("Asset is already checked out");
```

Comment:
I think this part was removed in a different branch and then merged. Make sure you watch the merge conflicts carefully. Remove this functionality, as it no longer does anything.